annotation-client/src/main/java/eu/europeana/annotation/client/connection/AnnotationApiConnection.java

```diff
@@ -419,7 +427,13 @@ public AnnotationOperationResponse createEuropeanaAnnotationLd(String motivation
 	/**
 	 * Execute Europeana API request
 	 */
-	return putURL(url, updateAnnotation, authorizationHeaderName, regularUserAuthorizationValue);
+	if (user!=null && user.equals(USER_ADMIN)) {
```

Comment:
please replace this implementation as indicated in the previous comment

Reply:
Done.